test/apiv2/python/rest_api/test_v2_0_0_image.py

```diff
@@ -86,6 +86,16 @@ def test_pull(self):
         self.assertTrue(keys["id"], "Expected to find id stanza")
         self.assertTrue(keys["images"], "Expected to find images stanza")
         self.assertTrue(keys["stream"], "Expected to find stream progress stanza's")
+    
+    def test_create(self):
+        r = requests.post(self.uri("/images/create?fromImage=alpine&message=testing123&platform=/linux[/amd64[/v8]]"), timeout=15)
```

Comment:
```suggestion
        r = requests.post(self.uri("/images/create?fromImage=alpine&message=testing123&platform=linux/amd64/v8"), timeout=15)
```
The brackets should not be sent.